parkhangs-webapp/src/features/modal/ModalDetail.js

```diff
@@ -61,56 +67,101 @@ class ModalDetail extends React.Component {
+                                        <b>Ends at:</b>
+                                    </td>
+                                    <td>
+                                        <b>Delete</b>
+                                    </td>
+                                    <td>
+                                        <b>Edit</b>
+                                    </td>
+                                </tr>
+                                </thead>
+                                <tbody>
+
+                                {
+                                    this.state.eventsOnCurrentDate.map((event) => {
+                                        return <tr key={event._id}>
+                                            <td>
+                                                {this.getCreatedTime(event.createdDateTime)}
+                                            </td>
+                                            <td>
+                                                {event.creatorName}
+                                            </td>
+                                            <td>
+                                                {event.details}
+                                            </td>
+                                            <td>
+                                                {this.getEventTime(event.eventDateTime)}
+                                            </td>
+                                            <td>
+                                                {
+                                                    this.getEventTime(event.eventEndDateTime) != null ?
+
+                                                        <div className="event-datetime">
+                                                            {this.getEventTime(event.eventEndDateTime)}
+                                                        </div>
+                                                        :
+                                                        null
+                                                }
+                                            </td>
+                                            <td>
+                                                <button onClick={() => this.editEvent(event)}>Edit</button>
+
+                                            </td>
+                                            <td>
+                                                <button onClick={() => this.deleteEvent(event._id)}>delete</button>
+                                            </td>
+                                        </tr>
+                                    })
+
+                                }
+
+                                </tbody>
+
+                            </table>
                             }
+
                         </div>
-                        {
-                            this.props.events.length === 0 &&
-                            <span className="EventsList-empty">There are no events on this date</span>
```

Comment:
does the table look okay when empty?

Reply:
I added text to show when theres no events